app/assets/stylesheets/flights.scss

```diff
@@ -1,3 +1,161 @@
-// Place all the styles related to the flights controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+// Style for will pagination
+.digg-pagination {
+  background: white;
+  cursor: default;
+
+  a, span, em {
+    padding: 0.2em 0.5em;
```

Comment:
Properties should be ordered display, float, margin-right, padding<br>Each selector in a comma sequence should be on its own single line